src/Markup/Perspex.Markup.Xaml/Converters/TimeSpanTypeConverter.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) The Perspex Project. All rights reserved.
+// Licensed under the MIT license. See licence.md file in the project root for full license information.
+
+using System;
+using System.Globalization;
+using OmniXaml.TypeConversion;
+using Perspex.Media;
+
+namespace Perspex.Markup.Xaml.Converters
+{
+    public class TimeSpanConverter : ITypeConverter
+    {
+        public bool CanConvertFrom(IXamlTypeConverterContext context, Type sourceType)
+        {
+            return sourceType == typeof(string);
+        }
+
+        public bool CanConvertTo(IXamlTypeConverterContext context, Type destinationType)
+        {
+            return false;
+        }
+
+        public object ConvertFrom(IXamlTypeConverterContext context, CultureInfo culture, object value)
+        {
+            // TimeSpan does not parse seconds format directly, restrict
+            // syntax to seconds for now (ie. "0.25")
+            var secs = double.Parse((string)value);
```

Comment:
I'd prefer to keep Duration as a `TimeSpan` as it's self-documenting. Agreed on the seconds format being shorter and making more sense in this situation though. Only consideration is that it might be confusing to someone coming from WPF/UWP etc but if that turns out to be the case maybe we can accept both.